package.json

```diff
@@ -6,7 +6,7 @@
     "build": "vue-cli-service build",
     "open-cypress": "TZ=UTC cypress open",
     "lint": "vue-cli-service lint --no-fix",
-    "build-bundle": "vue-cli-service build --target lib --name modeler ./src/components/nodes/index.js",
+    "build-bundle": "vue-cli-service build --modern --target lib --name modeler ./src/components/nodes/index.js",
```

Comment:
I'm not totally sure this is doing anything for us yet. I'll dig a little more into this.

Reply:
I wrote this comment up but looks like I forgot to include it in the review and it was lost:

I looked into the docs and it seems that in order to take advantage of modern mode, we'd have to include the appropriate `module` and `nomodule` scripts where the app is loaded.

However, when I run the build, I get the following warning:

>  WARN  Modern mode only works with default target (app). For libraries or web components, use the browserslist config to specify target browsers.

And it seems the same files are generated whether or not the `--modern` flag is set. So unless we significantly change our build process (switch from a lib build target to the default app build target), this change may only cause some confusion.